x/evm/precompile/test/export.go

```diff
@@ -173,14 +158,38 @@ func AssertWasmCounterState(
 		}
 `)
 
+	resp, err := deps.App.WasmKeeper.QuerySmart(deps.Ctx, wasmContract, msgArgsBz)
+	s.Require().NoError(err)
+	s.Require().NotEmpty(resp)
+
+	var typedResp QueryMsgCountResp
+	s.NoError(json.Unmarshal(resp, &typedResp))
+
+	s.EqualValues(wantCount, typedResp.Count)
+	s.EqualValues(deps.Sender.NibiruAddr.String(), typedResp.Owner)
+}
+
+func AssertWasmCounterStateWithEvm(
```

Comment:
One way of asserting the counter via wasmKeeper directly, and one way of asserting the counter via EVM wasm precompile call.